dB/postgresSchema.sql

```diff
@@ -0,0 +1,58 @@
+DROP DATABASE IF EXISTS similarproducts;
```

Comment:
Are you creating this manually? If so, I suggest having a script that allows you to programmatically update (see https://github.com/Yang-tze/ben-details/blob/master/data/generation/postgresGenerator.js for an example).